config/polygon.json

```
@@ -0,0 +1,27 @@
+{
+    "network": "matic",
+    "erc721": [
+        {
+            "name": "AavegotchiDiamond",
+            "address": "0x86935f11c86623dec8a25696e1c19a8659cbf95d",
+            "startBlock": 11516320
+        },
+        {
+            "name": "RealmDiamond",
+            "address": "0x1D0360BaC7299C86Ec8E99d0c1C9A95FEfaF2a11",
+            "startBlock": 20667840
+        },
+        {
+            "name": "ChronosTravaler",
+            "address": "0xa03c4e40d1fcaa826591007a17ca929ef8adbf1c",
+            "startBlock": 36796276
+        }
+    ],
+    "erc1155": [
+        {
+            "name": "WearableDiamond",
```

Let's always use the name of the game in the name of the NFT
```suggestion
            "name": "AavegotchiWearable",
```